modules/support/client/components/ReportMemberLink.component.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * A link to report member
+ * @param {string} username - the username of the user to report
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import '@/config/client/i18n';
+import { useTranslation } from 'react-i18next';
+
+export default function ReportMemberLink({ username }) {
+  const { t } = useTranslation('support');
+
+  return (
+    <span title={t('Report member {{username}} to support', { username })}>
```

Comment:
Why not just add this `title` to `a` tag?

Reply:
Oh I see, you're following the rendered output from Angular:
```
<small ng-if="::app.user._id !== profileCtrl.profile._id" tr-report-member="::profileCtrl.profile.username" title="Report member dalpus to support"><a ui-sref="support({report: username})" class="text-muted" href="/support?report=dalpus">Report member</a></small>
```

That's just unfortunate how directives work. We can make it better in React. :-)

Reply:
removed unnecessary tags and moved `title` => `aria-label` in 2424f57fd4d27e9edee34bbe6fd34af987b5bb42